js/narrativeView.js

```diff
@@ -308,12 +310,19 @@ define([
       this.model.setActiveItem(index);
     }
 
+    // In mobile view, highlight instruction if user navigates to another
+    // item before completing, in case the strapline is missed
```

Comment:
method/function comments are typically done like this:
```suggestion
  /**
   * In mobile view, highlight instruction if user navigates to another
   * item before completing, in case the strapline is missed
   */
```

this is so they automatically get picked up by your code editor as being documentation for that method:
![image](https://user-images.githubusercontent.com/1249597/111391760-2bf6f080-86ad-11eb-9bc4-fe7842dfcc28.png)

You can also use [JSDoc](https://jsdoc.app/) in these sorts of comments to describe the parameters that should be passed to the method and what sort of data (if any) it returns.

Reply:
Nice feature, one I didn't know about. I'll also look into JSDoc.